src/Promenade.Web/Views/Shared/_Layout.cshtml

```diff
@@ -254,11 +301,25 @@
                                             @foreach (var subItem in item.Navigations)
                                             {
                                                 <li>
-                                                    <a href="@subItem.Link"
-                                                       @(subItem.TargetNewWindow ? "target=\"_blank\"" : "")
-                                                       aria-label="@subItem.NavigationText.AriaLabel @(subItem.TargetNewWindow ? " (opens new window)" : "")">
-                                                        @subItem.NavigationText.Label
-                                                    </a>
+                                                    @if (string.IsNullOrEmpty(subItem.NavigationText.Label)
+                                                        && !string.IsNullOrEmpty(subItem.Icon)
+                                                        && !string.IsNullOrEmpty(subItem.NavigationText.Title))
+                                                    {
+                                                        <a href="@subItem.Link"
+                                                           @(subItem.TargetNewWindow ? "target=\"_blank\"" : "")
+                                                           title="@subItem.NavigationText.Title"
+                                                           aria-label="@(subItem.TargetNewWindow ? "(opens new window)" : "")">
+                                                            @subItem.NavigationText.Label
```

Comment:
The label text here can be removed and the icon added in its place.

Reply:
Resolved in commit 9b9fd46116bc0d3399fe1289b3f3852d4fd5576c